lib/rules/no-empty-buttons-v9.js

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+"use strict";
+
+const { hasTextContentChild } = require("../util/hasTextContentChild");
+var elementType = require("jsx-ast-utils").elementType;
+var hasProp = require("jsx-ast-utils").hasProp;
+
+//------------------------------------------------------------------------------
+// Rule Definition
+//------------------------------------------------------------------------------
+const allowedComponents = ["Button", "ToggleButton", "SplitButton", "MenuButton", "CompoundButton"];
+
+module.exports = {
+    meta: {
+        // possible error messages for the lint rule
+        messages: {
+            noEmptyButtons: "Accessibility: no empty buttons"
```

Comment:
For more clarity, we could make this a string template and print out the components at the end like this: ${allowedComponents.join(", ")}